.gitlab-ci.yml

```diff
@@ -0,0 +1,107 @@
+image: smk762/qt-ci:latest
+
+stages:
+  - build
+  - upload_zip
+  - upload_portable
+  - test
+  
```

Comment:
It's probably miss the approach of export env variable for branded dex -> https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/7d52326ac0780d6d9e192085776489114b11f5e8/.github/workflows/atomicdex-desktop-ci.yml#L12

(this one is different for each branded dex branch and it's mandatory)

Reply:
Will tackle this in a later PR.